persistence/scrobble_buffer_repository_test.go

```diff
@@ -0,0 +1,209 @@
+package persistence
+
+import (
+	"context"
+	"time"
+
+	"github.com/Masterminds/squirrel"
+	"github.com/navidrome/navidrome/log"
+	"github.com/navidrome/navidrome/model"
+	"github.com/navidrome/navidrome/model/request"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("ScrobbleBufferRepository", func() {
+	var scrobble model.ScrobbleBufferRepository
+	var rawRepo sqlRepository
+
+	enqueueTime := time.Date(2025, 01, 01, 00, 00, 00, 00, time.Local)
+
+	var insertManually = func(service, userId, mediaFileId string, playTime time.Time) {
+		ins := squirrel.Insert("scrobble_buffer").SetMap(map[string]interface{}{
+			// This _should_ be short enough such that there will not be a collision
+			// with a randomly generated ID. If ID generation changes, maybe update this
```

Comment:
Sorry, but wdym?

Reply:
If `id.NewRandom()` changes to be the same length as this string, that'd be a problem